var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -62,47 +72,53 @@ def install(self, spec, prefix):
             '-DCMAKE_Fortran_FLAGS=-fPIC',
             '-DEXAMPLES_ENABLE=ON',
             '-DEXAMPLES_INSTALL=ON',
-            '-DFCMIX_ENABLE=ON'
+            '-DFCMIX_ENABLE=ON',
+            '-DMPI_ENABLE=%s' % self.on_off('+mpi'),
+            '-DLAPACK_ENABLE=%s' % self.on_off('+lapack'),
+            '-DKLU_ENABLE=%s' % self.on_off('+klu'),
+            '-DHYPRE_ENABLE=%s' % self.on_off('+hypre'),
+            '-DSUPERLUMT_ENABLE=%s' % self.on_off('+superlu'),
+            '-DOPENMP_ENABLE=%s' % self.on_off('+openmp'),
+            '-DPTHREAD_ENABLE=%s' % self.on_off('+pthread')
```

Comment:
Another remark on style: I 'd prefer to remain with the same string substitution mechanism within the same file. Can we change these to something like:
```python
-DLAPACK_ENABLE={0}'.format(on_off('+lapack'))
```
?